controllers/operands/kubevirt.go

```diff
@@ -291,6 +293,20 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
+func filterWorkloadupdatemethods(workloadUpdateMethods []string) []string {
+	var filterWorkloadUpdateMethods []string
```

Comment:
please rename. Same name as the function - I find it confusing

Reply:
done